modules/renter.go

```diff
@@ -119,8 +119,9 @@ type HostDBEntry struct {
 
 // HostDBScan represents a single scan event.
 type HostDBScan struct {
-	Timestamp time.Time `json:"timestamp"`
-	Success   bool      `json:"success"`
+	Timestamp   time.Time         `json:"timestamp"`
```

Comment:
@DavidVorick is there still a need to keep the Timestamp?

Reply:
I'm going to go ahead and say yes, we should keep the timestamp in the scan, just in case we find some use for it later. We may (or may not) phase it out eventually, but I can think of a few possible reasons we may want it at some point.